realtime/src/main/java/com/mx/path/service/connection/realtime/MdxRealtimeConnectionConfiguration.java

```diff
@@ -1,14 +1,24 @@
 package com.mx.path.service.connection.realtime;
 
+import com.mx.path.gateway.configuration.annotations.ClientID;
 import lombok.Data;
 
 import com.mx.path.core.common.configuration.ConfigurationField;
 
 @Data
 public class MdxRealtimeConnectionConfiguration {
-  @ConfigurationField(value = "clientId")
+  @ConfigurationField
   private String clientId;
 
-  @ConfigurationField(value = "apiKey", secret = true)
+  @ConfigurationField(secret = true)
   private String apiKey;
+
+  @ConfigurationField(secret = true)
+  private String hmacSalt;
+
+  @ConfigurationField
+  private String hmacDigest;
```

Comment:
I don't think this is the right field name. Do you mean hmacKey or hmacSecret ? I think hmacDigest refers to the hashed signature for an individual API call, not something that would be in the configuration.

Maybe I'm missing something.